README.md

```diff
@@ -1,68 +1,61 @@
-  ```
-
-  Save the response somewhere.
-8. Copy the public key you received in the response and replace the contents of the file `smartthings_rsa.pub` with it. Be sure to replace all `\n` and `\r\n` with actual line breaks.
-9. Update the registered app OAuth (this defines the specific [permissions](https://smartthings.developer.samsung.com/develop/guides/smartapps/auth-and-permissions.html) your SmartApp may request). The `appId` can be found in the response when registering your SmartApp:
-  ```
-  curl -X "PUT" "https://api.smartthings.com/v1/apps/YOUR-APP-ID/oauth" \
-       -H "Authorization: Bearer YOUR-SMARTTHINGS-AUTH-TOKEN" \
-       -H "Content-Type: application/json; charset=utf-8" \
-       -d $'{
-    "scope": [
-      "r:devices:*",
-      "x:devices:*",
-      "r:schedules",
-      "w:schedules"
-    ],
-    "clientName": "Weather App OAuth Client"
-  }'  
-  ```
-10. Stop the server: `CTRL-C`.
-11. Start the server again: `npm start` (this ensures the public key will be used to verify requests from SmartThings).
-12. Install the SmartApp in the SmartThings mobile app (go to Marketplace->SmartApps->My Apps->Example Weather Color Light.
-13. Enter all required inputs on the configuration screens.
-14. Once installed, the configured bulb should turn on and its color should either be purple (if precipitation is in the forecast), orange (if the forecast calls for temperatures above 80 degrees Fahrenheit), blue (if the forecast calls for temperatures below 50 degrees Fahrenheit), or white (if no precipitation and temperature between 50 and 80 degrees Fahrenheit). It will check the current weather at the interval selected by the user during installation.
+
+2. Create an API key with [Open Weather Map](https://api.openweathermap.org) (free tier is fine), and store it in an environment variable named `WEATHER_API_KEY`.
+
+3. Install the dependencies for this app: `npm install`.
+
+4. Start the server: `npm start`.
+
+5. Start ngrok (in another terminal window/tab): `ngrok http 3005`. Copy the `https:` URL to your clipboard.
+
+6. Go to the [Automation](https://devworkspace.developer.samsung.com/smartthingsconsole/iotweb/site/index.html#/development/automation) section of the Developer Workspace and create an Automation.
+	- For the **SmartApp Type** select **WebHook endpoint** and enter the https URL you copied from the above step.
+	- For the **Scope**, click on **Settings** and select the following scopes:
+		- `r:devices:*`,
+		- `x:devices:*`,
+		- `r:schedules`, and
+		- `w:schedules`
+	- Click **SAVE AND NEXT**.
+	- In the next screen you will be presented with the **Public Key**.
+
+7. Copy this public key and replace the contents of the file `config/smartthings_rsa.pub` with it. Be sure to replace all `\n` and `\r\n` with actual line breaks.
```

Comment:
I think the key shown in the workspace doesn't need any newline char replacements. Can just be used as-is

Reply:
True. I did not see the escape characters in dev Workspace window. However, to be sure, I did run into `forbidden - failed verifySignature` error and I suspected some editor issue, switched back and forth from Atom and Sublime text. Separate issue though, I think.